rust/arrow/src/array/array.rs

```diff
@@ -2592,6 +2619,15 @@ mod tests {
         assert_eq!(DataType::Int32, list_array.value_type());
         assert_eq!(3, list_array.len());
         assert_eq!(0, list_array.null_count());
+        assert_eq!(
```

Comment:
The rest of the tests were passing previously. This was the failing test. The actual value returned was 0.